Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +162,25 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        if self.LOSS_FUNCTIONS[self.loss_function] != "Squared loss":
```

Comment:
Do not compare string literals. Use:

    if self.loss_function == SqLoss:

Also elsewhere.